knox/auth.py

```diff
@@ -6,20 +6,16 @@ def compare_digest(a, b):
 
 import binascii
 
-from django.utils.translation import ugettext_lazy as _
-from django.utils import timezone
 from django.contrib.auth import get_user_model
-
-from rest_framework import exceptions
-from rest_framework.authentication import (
-    BaseAuthentication,
-    get_authorization_header
-)
-
+from django.utils import timezone
+from django.utils.translation import ugettext_lazy as _
 from knox.crypto import hash_token
 from knox.models import AuthToken
 from knox.settings import CONSTANTS, knox_settings
 from knox.signals import token_expired
+from rest_framework import exceptions
+from rest_framework.authentication import (BaseAuthentication,
+                                           get_authorization_header)
```

Comment:
Can we configure isort so that it gives:
```
from rest_framework.authentication import (	
    BaseAuthentication,	
    get_authorization_header,
)
```

Reply:
Yes my friend, consider it done :)